test/lib/testHelpers.ts

```diff
@@ -87,6 +89,24 @@ export const advanceToNextEpoch = async (epochManager: EpochManager): Promise<vo
 export const evmSnapshot = async (): Promise<number> => provider().send('evm_snapshot', [])
 export const evmRevert = async (id: number): Promise<boolean> => provider().send('evm_revert', [id])
 
+// Math
+
+export const MAX_PPM = toBN('1000000')
+
+export function weightedAverage(
+  valueA: BigNumber,
+  valueB: BigNumber,
+  weightA: BigNumber,
+  weightB: BigNumber,
```

Comment:
It might help to make this function exactly the same params as the solidity function:

```
    function weightedAverage(
        uint256 valueA,
        uint256 weightA,
        uint256 valueB,
        uint256 weightB
    ) 
```
    

Could prevent future bugs

Reply:
Good idea, I'll reorder the input values in the JS function